metals/src/main/scala/scala/meta/internal/metals/JavaInteractiveSemanticdb.scala

```diff
@@ -180,7 +181,8 @@ object JavaInteractiveSemanticdb {
   def create(
       javaHome: AbsolutePath,
       workspace: AbsolutePath,
-      buildTargets: BuildTargets
+      buildTargets: BuildTargets,
+      maybeJDKVersion: Option[JdkVersion]
```

Comment:
```suggestion
      jdkVersion: JdkVersion
```
I think this shouldn't be an option.
In `MetalsLanguageServer` we infer a version - it's it's none then there is no point in having `JavaInteractiveSemanticdb`

I noticed that only here there is a call of `JdkVersion.getJavaVersionFromJDK8FallBack` but probably it should be included into a method that infers version in `MetalsLanguageServer`

Reply:
is the fallback to `JdkVersion.getJavaVersionFromJDK8FallBack` useful also in `ProblemSolver`? because that is the other place in MetalsLanguageServer in whicch `maybeJdkVersion` is used.

Reply:
> is the fallback to JdkVersion.getJavaVersionFromJDK8FallBack useful also in ProblemSolver

I've just realized that the right thing would be to remove `getJavaVersionFromJDK8FallBack`.
Anyway it won't be called because the version should be obtained from shell.